must-gather/collection-scripts/debug/tail_failed_pods

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+skip_tls="${1:-false}"
+
+for pod in $(oc get pods --insecure-skip-tls-verify=${skip_tls} -n openshift-adp -o jsonpath='{.items[?(.status.containerStatuses[0].lastState.terminated.reason=="Error")].metadata.name}'); do
+    echo "***"
+    echo "* Last logs from failed pod openshift-adp/${pod}:"
+    oc logs -n openshift-adp $pod --tail=10
```

Comment:
Do you think we need insecure here too?

Reply:
At some point probably all of this must-gather should be reviewed for this, there are a bunch of places that don't look at skip_tls at all.